src/Facebook/FacebookBatchResponse.php

```diff
@@ -92,7 +92,7 @@ public function setResponses(array $responses)
     /**
      * Add a response to the list.
      *
-     * @param int        $key
+     * @param mixed      $key
```

Comment:
Can we just do `int|string` here too? Why `mixed`? Would `$key` be anything else other than int or string? :)